sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_mcps.c

```diff
@@ -358,7 +373,14 @@ void gnrc_lorawan_event_no_rx(gnrc_lorawan_t *mac)
         return;
     }
 
-    /* Otherwise check if retransmission should be handled */
```

Comment:
why did you remove this comment?

Reply:
Oops. I guess this got removed due to a merge conflict while rebasing.

Reply:
This got removed in PR [15946](https://github.com/RIOT-OS/RIOT/pull/15946) . After rebasing on top of it it got removed.